.github/ISSUE_TEMPLATE/docs.md

```diff
@@ -0,0 +1,31 @@
+---
+name: 'Drift docs: Issue / Feature request'
+about: Create a report to help improve drift's docs
+title: ''
+labels: ['docs']
+assignees: ''
+---
+
+**Description**
+
+Thanks for taking the time to file an issue for drift's docs!
+
+For feature requests, please describe the feature you're interested in.
+
+For information requests, please describe what you think is missing from
+the docs and if possible where should this information be placed.
```

Comment:
Hopefully we'll have a complete docs site after the switch to MkDocs and users won't have to ask for basic things like a copy button for code snippets :D So let's merge these paragraphs into one:

```suggestion
If you are looking for information that you feel is not sufficiently explained on the
website (or too hard to find), please describe what the site is missing and, if possible,
where you think the information should be added.
```